### DISCUSSION TITLE
Dumb questions about Budgie 11 that I have.

### DISCUSSION THREAD

**Discussion by BrazilianTankie** [BuddiesOfBudgie/budgie-desktop#549] [Q&A] [answered]
Heya I am using Budgie Desktop and it's great! Really great experience but I have some questions and didn't found anywhere.

All these questions are GTK or Gnome Online Accounts related.

1. Is GTK 3/4 themes going to continue working in Budgie 11?

2. Is gnome online accounts going to stay in Budgie 11?

3. Will the gnome store stay in Budgie 11?

**Comment by serebit** [comment, marked as answer]
1. Yes, but only for third-party GTK3 and GTK4 apps. Budgie itself will not be using GTK in version 11.
2. No.
3. It should be usable, as it is in version 10.

**Reply by BrazilianTankie** [reply]
Thanks!
